examples/github-deploy-only.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request, release]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.4.2
+
+    - name: Install Visual C++ for Python 2.7
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        choco install vcpython27 -f -y
+
+    - name: Build wheels
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
+
+    - uses: actions/upload-artifact@v1
+      with:
+        name: wheels
+        path: ./wheelhouse
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Build sdist
+      run: python setup.py sdist -d wheelhouse/
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: wheelhouse/*.tar.gz
```

Comment:
```suggestion
        path: dist/*.tar.gz
```

Reply:
Might as well leave this default too; the upload/download will put all these together for you anyway.

Reply:
Does this work if the wheels are still built in `wheelhouse`?

Reply:
Sure. This only uploads the dist.